### PR TITLE
Fix crash

### PR DESCRIPTION
**Pull request by witcpalek**

### CODE REVIEW

File: src/main/cpp/mechanisms/MechanismFactory.h [hunk 1]
```diff
@@ -81,9 +81,9 @@ class MechanismFactory
 		
 		
 		// @ADDMECH  Add inline Get method for your mechanism
-		inline Arm* Getarm() const {return m_arm;}
-		inline Extender* Getextender() const {return m_extender;}
-		inline Grabber* Getgrabber() const {return m_grabber;}
+		inline Arm* GetArm() const {return m_arm;}
+		inline Extender* GetExtender() const {return m_extender;}
+		inline Grabber* GetEGrabber() const {return m_grabber;}
```
Comment (nickfrisch) [inline src/main/cpp/mechanisms/MechanismFactory.h]: Fix EGrabber to Grabber

File: src/main/cpp/mechanisms/grabber/GrabberState.cpp [hunk 2]
```diff
@@ -44,8 +44,8 @@
 		,
 MechanismTargetData::SOLENOID solState0,
 MechanismTargetData::SOLENOID solState1
-		) : Mech2SolenoidsState( MechanismFactory::GetMechanismFactory()->Getgrabber(), stateName, stateId , solState0, solState1),
-		m_grabber(MechanismFactory::GetMechanismFactory()->Getgrabber())
+		) : Mech2SolenoidsState( MechanismFactory::GetMechanismFactory()->GetEGrabber(), stateName, stateId , solState0, solState1),
```
Comment (nickfrisch) [inline src/main/cpp/mechanisms/grabber/GrabberState.cpp]: Switch EGrabber to Grabber

File: src/main/cpp/mechanisms/grabber/GrabberState.h [hunk 3]
```diff
@@ -47,7 +47,7 @@ MechanismTargetData::SOLENOID solState1
 
 
 		bool AtTarget() const override;
-		Grabber * GetGrabber() const {return m_grabber;}
+		Grabber * GetEGrabber() const {return m_grabber;}
```
Comment (nickfrisch) [inline src/main/cpp/mechanisms/grabber/GrabberState.h]: Switch EGrabber to Grabber

File: src/main/cpp/mechanisms/grabber/GrabberStateMgr.cpp [hunk 4]
```diff
@@ -46,7 +46,7 @@ GrabberStateMgr* GrabberStateMgr::GetInstance()
 {
 	if ( GrabberStateMgr::m_instance == nullptr )
 	{
-        auto grabberPtr = MechanismFactory::GetMechanismFactory()->Getgrabber();
+        auto grabberPtr = MechanismFactory::GetMechanismFactory()->GetEGrabber();
```
Comment (nickfrisch) [inline src/main/cpp/mechanisms/grabber/GrabberStateMgr.cpp]: Switch EGrabber to Grabber

File: src/main/cpp/mechanisms/grabber/GrabberStateMgr.cpp [hunk 5]
```diff
@@ -59,7 +59,7 @@ GrabberStateMgr* GrabberStateMgr::GetInstance()
 
 /// @brief    initialize the state manager, parse the configuration file and create the states.
 GrabberStateMgr::GrabberStateMgr() : StateMgr(),
-                                     m_grabber(MechanismFactory::GetMechanismFactory()->Getgrabber())
+                                     m_grabber(MechanismFactory::GetMechanismFactory()->GetEGrabber())
```
Comment (nickfrisch) [inline src/main/cpp/mechanisms/grabber/GrabberStateMgr.cpp]: Switch EGrabber to Grabber

File: src/main/cpp/mechanisms/MechanismFactory.h (hunk 1 above)
Comment (witcpalek) [reply src/main/cpp/mechanisms/MechanismFactory.h]: Updated CP. thanks

File: src/main/cpp/mechanisms/grabber/GrabberState.h (hunk 3 above)
Comment (witcpalek) [reply src/main/cpp/mechanisms/grabber/GrabberState.h]: Updated thanks

File: src/main/cpp/mechanisms/grabber/GrabberStateMgr.cpp (hunk 5 above)
Comment (witcpalek) [reply src/main/cpp/mechanisms/grabber/GrabberStateMgr.cpp]: Updated thanks

**Review by nickfrisch — APPROVED** [review]

**Pull request merged**